dali/operators/segmentation/random_object_bbox.cc

```diff
@@ -162,6 +162,38 @@ void RandomObjectBBox::FindLabels(std::unordered_set<int> &labels, const T *data
   }
 }
 
+
+template <typename BlobLabel, typename T>
+void RandomObjectBBox::FindLabels(SampleContext<BlobLabel> &ctx,
+                                  const TensorView<StorageCPU, const T> &in) {
+  ctx.labels.clear();
+  int64_t N = in.num_elements();
+  if (!N)
```

Comment:
```suggestion
  if (N == 0)
```
might be just my personal preference. Feel free to ignore